pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+
+
+....................................................
+Before you submit a pull request, please make sure :
```

Comment:
Isn't a template supposed to be some skeleton where you will in the blanks?

From this perspective, the content of this file should probably be

- Brief description:
- link to a presentation in a [fill the blank] group:
- expected changes:

... and then a few things to suggest compliance. Please edit as appropriate:
- I did **not** check compliance with the code rules
- I did **not** run tests before submitting the PR